x-pack/legacy/plugins/infra/public/pages/logs/analysis/setup/initial_configuration_step/analysis_setup_indices_form.tsx

```diff
@@ -53,20 +74,17 @@ export const AnalysisSetupIndicesForm: React.FunctionComponent<{
         />
       }
     >
-      <EuiFormRow
-        describedByIds={['indices']}
-        error={validationErrors.map(formatValidationError)}
-        fullWidth
-        isInvalid={validationErrors.length > 0}
-        label={indicesSelectionLabel}
-        labelType="legend"
-      >
-        <EuiCheckboxGroup
-          options={choices}
-          idToSelectedMap={indices}
-          onChange={handleCheckboxGroupChange}
-        />
-      </EuiFormRow>
+      <LoadingOverlayWrapper isLoading={isValidating}>
+        <EuiFormRow
+          describedByIds={['indices']}
+          fullWidth
+          isInvalid={!valid}
+          label={indicesSelectionLabel}
+          labelType="legend"
+        >
+          <>{choices}</>
```

Comment:
`choices` is an array here, but `EuiFormRow` only takes one child. Using a fragment tricks the component into believing there's only one node.